components/shared/script/lib.rs

```diff
@@ -248,6 +248,8 @@ pub enum ScriptThreadMessage {
     /// Evaluate the given JavaScript and return a result via a corresponding message
     /// to the Constellation.
     EvaluateJavaScript(PipelineId, JavaScriptEvaluationId, String),
+    /// Parameter indicates if online (true) or offline (false)
+    SetNetWorkState(bool),
```

Comment:
Nothing ever sends this message... how is it expected to be sent?

```suggestion
    /// Parameter indicates if online (true) or offline (false)
    SetNetworkOnlineState(bool),
```